mitosheet/src/mito/components/taskpanes/Graph/GraphExportTab.tsx

```diff
@@ -52,7 +51,7 @@ function GraphExportTab(
                 <TextButton
                     variant='dark'
                     onClick={copyShowGraphCode}
-                    disabled={props.loading || props.graphOutput === undefined}
+                    disabled={props.graphOutput === undefined}
```

Comment:
I think we need to keep loading here, yes? Or is it that loading is no longer accessible.